datalad/resources/website/assets/js/main.js

```diff
@@ -78,6 +78,22 @@ function getParameterByName(name, url) {
 }
 
 /**
+ * Create Breadcrumbs to current location in dataset
+ * @return {array} html linkified breadcrumbs array
+ */
+function bread2crumbs() {
+  var raw_crumbs = loc().href.split('/');
+  var crumbs = [];
+  for (var index = 2; index < raw_crumbs.length() - 1; index++) {
+    if (raw_crumbs[index] !== '?dir=')
+      continue;
```

Comment:
wanna see a webshot! ;)  a bit not sure about logic -- so it would show only dataset boundaries (because of `continue`) or somehow demarcate those boundaries (e.g. as we talking with `//` to start with?)